### PR TITLE
Ev/rotate docs

### PR DESCRIPTION
**Pull request by kontsevoy**
Documented the certificate rotation.

Notes:

*  #1912 must be merged first.

### CODE REVIEW

File: docs/2.5/admin-guide.md [hunk 1]
```diff
@@ -1448,6 +1448,59 @@ To allow access for all users:
   * Update `sshd` configuration (usually `/etc/ssh/sshd_config`) to point to this
   file: `TrustedUserCAKeys /etc/ssh/teleport-user-ca.pub`
 
+## Certificate Rotation
+
+Take a look at the [Certificates chapter](architecture.md#certificates) in the
+architecture document to learn how the certificate rotation works. This section
+will show you how to implement certificate rotation in practice.
```
Comment (klizhentas) [inline docs/2.5/admin-guide.md]: it's not only certificate rotation, but it's certificate authority rotation - which means CA has been rotated as well, I think it's an important enough distinction to point it out here.

Comment (kontsevoy) [reply docs/2.5/admin-guide.md]: yes, that's why I moved all of this into "how it works" section in the "architecture", where I (for the first time, actually) explicitly stated that there are TWO CAs and they're rotated.

Comment (klizhentas) [reply docs/2.5/admin-guide.md]: k, cool

**Review by klizhentas — APPROVED** [review]

**Review by twakes — COMMENTED** [review]
These look good to me. One general comment that comes to mind here, although applies to other places in the documentation is that it would be nice to show sample responses to command issued so you know you are doing it right.

**Comment by klizhentas** [comment]
retest this please

**Comment by klizhentas** [comment]
retest this please

**Comment by klizhentas** [comment]
retest this please

**Pull request merged**